notebooks/example.ipynb

```diff
@@ -200,6 +199,143 @@
     "%%bash\n",
     "cat ./fixtures/rendered_config.yaml"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ea5eff92",
+   "metadata": {},
+   "source": [
+    "## Example 3: Comparing two config files\n",
+    "Let's explore using the at using the `config.compare()` method to compare two two config files"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "71d27692",
+   "metadata": {},
+   "source": [
+    "We again need to start by importing the `uwtools.api.config` from the uwtools python package."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 52,
+   "id": "7fb40514",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from uwtools.api import config"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b0103f44",
+   "metadata": {},
+   "source": [
+    "The uwtools.api.config.compare() method has the following syntax:\n",
```

Comment:
```suggestion
    "The `config.compare()` method has the following syntax:\n",
```
I think that, since we're doing `from uwtools.api import config`, we can thereafter ignore the `uwtools.api` part and treat `config` as the main namespace and concept, which cuts down on the verbosity a bit.

Reply:
Since we have this information already rendered nicely in our HTML docs -- here: https://uwtools.readthedocs.io/en/main/sections/user_guide/api/config.html#uwtools.api.config.compare -- would it work to refer the reader to that, both to avoid repeating ourselves (and having to keep the information in this notebook in-sync with changes that might occur later), and to familiarize the notebook reader with the official docs? We might say something like
> Please review the [`config.compare()` documentation](https://uwtools.readthedocs.io/en/main/sections/user_guide/api/config.html#uwtools.api.config.compare) for full information on this function's arguments.